mir_eval/io.py

```diff
@@ -581,3 +503,45 @@ def load_ragged_time_series(filename, dtype=float, delimiter=r'\s+',
             values.append(converted_value)
 
     return np.array(times), values
+
+
+def save_evaluation(results, output_file):
+    # TODO Use NumPy style docstrings like the rest of mir_eval.
+    '''
+    Write a result dictionary out as a .json file.
+
+    :parameters:
+        - results : dict
+            Results dictionary, where keys are metric names and values are
+            the corresponding scores
+        - output_file : str
+            Path to .json file to write to
+    '''
+    with open(output_file, 'w') as f:
+        json.dump(results, f)
+
+
+def print_evaluation(results):
```

Comment:
This function should go in util, not io.  It doesn't involve I/O.

Reply:
Arguable IMO as `sys.stdout` is treated as an always open file object in Python, no? But more importantly, it should be possible to remove the function entirely (maybe pandas instead?).